### DISCUSSION TITLE
Where can i find the installation guide?

### DISCUSSION THREAD

**Discussion by YesShubham01** [Homebrew/discussions#4084] [Everyday usage]
### Output of `brew config`

```shell
.
```

### Output of `brew doctor`

```shell
.
```

### Description of issue

Where can i get installations guide?

**Comment by SMillerDev** [comment]
It's literally the first thing on our website brew.sh

**Reply by YesShubham01** [reply]
How can i understand and work on the code? ( fixing issues )

**Reply by SMillerDev** [reply]
I don't think the discussion board is quite the place to ask for help how to become a developer. Here is the contributor guid though: https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md
